clients/ui/frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18 AS build-stage
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+
+RUN npm run build
+
+FROM nginxinc/nginx-unprivileged
+
+ENV API_URL="http://localhost:4001"
```

Comment:
I would add this env param to the docker-compose and the future Deployment, to have ability to parametrize this in the future.

Reply:
This param is in the docker compose - it's being overridden to point to the bff via the hostname assigned by docker. This essentially just sets a default value.

Reply:
you're right, we could just match `/api/` and redirect that - that probably is cleaner. I used to prefer a visual clue that something's being proxied.